lib/ssl/doc/src/ssl.xml

```diff
@@ -393,9 +401,80 @@
 	</p>
       </desc>
     </datatype>
-    
+
+     
+      <datatype>
+	<name name="signature_algs"/>
+	<desc>
+
+	  <p>In TLS 1.2, signatures algorithms used for the TLS protocol messages
+	  during the handshake, can be negotiated separatly from the negotiated
+	  cipher suite using the TLS signature
+	  algorithm extension <url
+	  href="http://www.ietf.org/rfc/rfc5246.txt">Section 7.4.1.4.1 in RFC 5246</url>.
+	  The extension is defined a list of {<seetype marker="#hash">hash()</seetype>,  <seetype marker="#signature_algs">sign_algo()</seetype>} pairs.
+	  In TLS-1.3 cipher suite algorithms and these algorithms are compleatly decoupled and the list of possible values
+	  are defined by the <seetype marker="#sign_scheme">sign_scheme()</seetype> type. The algorithms should be in the preferred
+	  order. 
+	  
+	  TLS-1.2 default is </p>
+	  
+	  <code>[
+%% SHA2
+{sha512, ecdsa},
+{sha512, rsa},
+{sha384, ecdsa},
+{sha384, rsa},
+{sha256, ecdsa},
+{sha256, rsa},
+{sha224, ecdsa},
+{sha224, rsa},
+%% SHA
+{sha, ecdsa},
+{sha, rsa},
+{sha, dsa}
+]
+</code>
+
+	 <p>TLS-1.3 default is</p>
+
+	 <code>[
+ecdsa_secp521r1_sha512,
+ecdsa_secp384r1_sha384,
+ecdsa_secp256r1_sha256,
+rsa_pss_pss_sha512,
+rsa_pss_pss_sha384,
+rsa_pss_pss_sha256,
+rsa_pss_rsae_sha512,
+rsa_pss_rsae_sha384,
+rsa_pss_rsae_sha256,
+eddsa_ed25519,
+eddsa_ed448,
+%% Only applies to certificate signatures for backward compatible with TLS-1.2
+rsa_pkcs1_sha512,
+rsa_pkcs1_sha384,
+rsa_pkcs1_sha256,
+ecdsa_sha1,
+rsa_pkcs1_sha1
+]
+</code>
+
+            <p>If both TLS-1.3 and TLS-1.2 are supported the defaults will be merged.
+	    </p>
+	    
+	    
+	    <p> Support for {md5, rsa} was removed form the the TLS-1.2 default in ssl-8.0 (OTP-22) </p>
+	    
+	    <note> <p>TLS-1.2 algorithms will not be
+	    negotiated for TLS-1.3, but TLS-1.3 RSASSA-PSS  <seetype marker="#rsa_pss_scheme">rsa_pss_scheme()</seetype>
```

Comment:
```suggestion
	    negotiated for TLS-1.3, but TLS-1.3 RSASSA-PSS  <seetype marker="#rsassa_pss_scheme">rsassa_pss_scheme()</seetype>
```

Reply:
Thanks @garazdawi, had already fixed that one when I saw your comment. And now I hopefully fixed the last one.